results-mieb/openai/clip-vit-base-patch32/no_model_name_available/no_revision_available/model_meta.json

```diff
@@ -0,0 +1 @@
+{"name": "no_model_name_available", "revision": "no_revision_available", "release_date": null, "languages": null, "n_parameters": null, "memory_usage": null, "max_tokens": null, "embed_dim": null, "license": null, "open_source": null, "similarity_fn_name": null, "framework": [], "loader": null}
```

Comment:
Seems like CLIPModelWrapper does not use the ModelMeta object (see e.g. the e5 implementations).

Reply:
+ any metadata that we should update here?

Reply:
yeah, haven't got to this last time. Will gradually add

Reply:
have updated to use model meta. [model](https://github.com/embeddings-benchmark/mteb/blob/mieb/mteb/models/clip_models.py) [result](https://github.com/embeddings-benchmark/mteb/blob/mieb/results-mieb/openai__clip-vit-base-patch16/57c216476eefef5ab752ec549e440a49ae4ae5f3/RenderedSST2.json)